### PR TITLE
docs links updates

### PR DESCRIPTION
**Pull request by isunaslabs**
-Made docs links updates

### CODE REVIEW

File: src/data/menu.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ const getMenu = (t: TFunction<"translation">): Menu[] => {
     },
     {
       title: t(localeKeys.support),
-      path: "https://darwinia.notion.site/Darwinia-Support-c9412ead803542f197ba1ccafb72c427",
+      path: "https://darwinia.notion.site/a76d5bdc7ad242bea991c023dfca208f?v=5fb127fc9a7a4d67a3bbc3ed50445180",
```
Comment (fewensa) [inline src/data/menu.tsx:60]: It's look updated by Denny https://github.com/darwinia-network/home/commit/7b051abf7a1bda4d98b5f25e2b0c0ddcaec69276#diff-2a8e835c38b766de63259cb630fe9966ef9a84b21db7928a70e0b28e846143d3R60

**Comment by 1022ranji** [comment]
image
please update these links

File: src/data/menu.tsx (hunk 1 above)
Comment (1022ranji) [reply src/data/menu.tsx:60]: @isunaslabs Please use this: https://www.notion.so/darwinia/Darwinia-Support-c9412ead803542f197ba1ccafb72c427
I will update the content of this page later.

Comment (isunaslabs) [reply src/data/menu.tsx:60]: Since the changes aren't complete I think we can simply just merge with mine. There will be no conflict

**Review by 1022ranji — APPROVED** [review]

**Pull request merged**